packages/trpc/src/procedures/auth.ts

```diff
@@ -15,9 +15,11 @@ const getNewSessionExpirationDate = () =>
 export const signUpProcedure = notAuthenticatedProcedureBuilder
```

Comment:
This procedure will be redone in a future ticket to support the new login flow, just placeholder logic for now